app/src/main/java/org/wikipedia/travel/MainPlannerFragment.java

```diff
@@ -225,11 +239,11 @@ public boolean onBackPressed() {
      */
     private void saveTrip(Trip trip) {
         bSave.setActivated(false);
-        CallbackTask.execute(() -> TripDbHelper.instance().updateList(trip),  new CallbackTask.DefaultCallback<Object>(){
+        CallbackTask.execute(() -> TripDbHelper.instance().updateList(trip), new CallbackTask.DefaultCallback <Object>() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['<' is preceded with whitespace.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1526985)